docs/quickstart/quickstart-non-docker.md

```diff
@@ -54,6 +54,7 @@ connect is [UP]
 
     ```bash
     $ cd ksql
+    $ git checkout v0.3 -b 0.3
```

Comment:
should this be `git checkout v0.3-rc2 -b 0.3` ?

Reply:
No. I am using values for the final release. In the end the tag will be `v0.3`.